app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,39 @@
+/* universal */
+
+body {
+padding-top: 60px;
+}
+
+section {
+overflow: auto;
+}
+
+textarea {
+resize: vertical;
+}
+
+.center {
+text-align: center;
+}
+
+.center h1 {
+margin-bottom: 10px;
+}
+
+/* header */
+
+#logo {
+float: left;
```

Comment:
wrong indent

Reply:
for example 
![image](https://user-images.githubusercontent.com/56816494/130424973-80899fb8-0c2c-45d7-b890-f3d8960e1e00.png)